source/plugin/basics/blocks/index.rst

```diff
@@ -0,0 +1,16 @@
+===================
+Working with Blocks
+===================
+
```

Comment:
I like the first paragraph of your overview. 

Suggestions: 

* Remove the headings you added
* Remove the second paragraph

Like this:

Working with Blocks

Blocks are the very foundation of every Minecraft world, the element that makes .... and how they can be tamed to behave exactly like you want them to.

Contents

...